packages/fxa-payments-server/src/store/utils.tsx

```diff
@@ -18,11 +18,23 @@ export const setStatic = (newState: object) => (state: object): object => ({
   ...newState,
 });
 
-export const fetchDefault = (defaultResult: any): FetchState<any> => ({
-  error: null,
-  loading: false,
-  result: defaultResult,
-});
+export function fetchDefault<T = any, E = any>(
```

Comment:
This upgrade to `fetchDefault` makes the function do multiple duties - one of the most important is that it can now also define the specific `FetchState` type for store properties at the same time as initializing them in `defaultState`. That removes the need for defining a separate `State` type.